pkg/s3signer/request-signature-v2.go

```diff
@@ -42,9 +42,7 @@ const (
 func encodeURL2Path(u *url.URL) (path string) {
 	// Encode URL path.
 	if isS3, _ := filepath.Match("*.s3*.amazonaws.com", u.Host); isS3 {
```

Comment:
Can this `*.s3*.amazonaws.com` be changed to `*.s3.amazonaws.com`, it is little confusing with the current style.